tests/TestCase/ORM/EntityTest.php

```diff
@@ -1183,30 +1183,30 @@ public function testErrors()
     {
         $this->deprecated(function () {
             $entity = new Entity();
-            $this->assertEmpty($entity->errors());
-            $this->assertSame($entity, $entity->errors('foo', 'bar'));
-            $this->assertEquals(['bar'], $entity->errors('foo'));
+            $this->assertEmpty($entity->getErrors());
+            $this->assertSame($entity, $entity->getErrors('foo', 'bar'));
```

Comment:
This should be a setter call `$entity->setErrors('foo', 'bar')`, that's what's causing tests to fail.